README.md

```diff
@@ -59,18 +62,18 @@ Finally, you will also have to make sure that you are ignoring files that need i
 
 ### Checkout dependencies
 
-App Inventor uses Blockly, the web-based visual programming editor from Google, as a core part of its editor. Blockly core is made available to App Inventor as a git submodule. The first time after forking or cloning the repository, you will need to perform the following commands:
+App Inventor uses the [Closure library](https://github.com/google/closure-library) and the [Picrin](https://picrin.readthedocs.io/en/latest/) Scheme implementation. It is unlikely that most contributors will need to make changes to these dependencies, but if so, you can initialize and track these libraries as submodules. The first time after forking or cloning the repository, you can perform the following commands:
```

Comment:
I'm not sure I agree with the language here. If one doesn't initialize the submodules the system will not compile, so this step is necessary regardless of the clause "it is unlikely that most contributors will need to make changes to these dependencies"

Reply:
Let me know if that’s any better! @ewpatton